templates/forms/resend-key.php

```diff
@@ -54,7 +54,10 @@
 HTML;
 	}
 
-	$message_above_input_field = fs_esc_html_inline( "Enter the email address you've used for the upgrade below and we will resend you the license key.", 'ask-for-upgrade-email-address', $slug );
+	$message_above_input_field = $fs->is_only_premium() ?
+        fs_esc_html_inline( "Enter the email address you've used during the purchase and we will resend you the license key.", 'ask-for-upgrade-email-address-premium-only', $slug ) :
```

Comment:
@xiahengchen, please add indentation here (4 spaces).

```php
$message_above_input_field = $fs->is_only_premium() ?
    fs_esc_html_inline( "..." ) :
    fs_esc_html_inline( "..." );
```

Also the file seems to have mixed indentation. Please change the indentation of the touched lines to 4 spaces (includes the tab before the variable `$message_above_input_field`.